app/core/models/ticket/markdown.py

```diff
@@ -0,0 +1,19 @@
+
+
+
+class TicketMarkdown:
+    """Ticket and Comment markdown functions
+    
+    Intended to be used for all areas of a tickets, projects and comments.
+    """
+
+
+    def render_markdown(self, markdown):
```

Comment:
ensure in all places where ticket related markdown is used, this function does the rendering, not the template tag.

required as this is where the pretty printing of items will be inserted.